services/src/tasks/mod.rs

```diff
@@ -127,7 +129,10 @@ impl ToSchema for TaskStatus {
                             .schema_type(SchemaType::String)
                             .enum_values::<[&str; 1], &str>(Some(["running"])),
                     )
-                    .build(),
+                    .property("info", Object::new())
+                    .property("pct_complete", Object::with_type(SchemaType::String))
```

Comment:
should be camelCase

Reply:
[50cb9a4](https://github.com/geo-engine/geoengine/pull/706/commits/50cb9a49684bc28cfcdb570d35aff44dbc5b39ee)